src/test/java/com/epam/tc/hw1/CalculatorTesting.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.BeforeMethod;
+
+public class CalculatorTesting {
```

Comment:
Why do you do not extend test classes from it?

Reply:
I'm goof -_-